docs/examples/react/README.md

```diff
@@ -0,0 +1,49 @@
+---
+id: react-example
+title: React
+---
+
+A sample React app for [KintoHub](https://kintohub.com)
+
+### :rocket: Deploying on KintoHub
+
+- **Service Type:** Static Site
+- **Repo:** https://github.com/kintohub/kintohub
+- **Branch:** master
+- **Language:** NodeJS
+- **Version:** 13
+- **Build:** `npm install && npm run build`
+- **Subfolder Path:** `./docs/examples/react`
+- **Build Output Path:** `build`
+
+This will take a few minutes to deploy. Then click on the **Access** tab and copy the `External Access` URL to see the React App.
+
+## :hammer: Running locally
+
+**Pre-requisites:** Node.js v13
```

Comment:
Node.js -> NodeJS. always use the same format

Reply:
Done